pkg/features/features.go

```diff
@@ -76,6 +76,12 @@ const (
 	// owner: @dimityrmirchev
 	// alpha: v1.93.0
 	ShootManagedIssuer featuregate.Feature = "ShootManagedIssuer"
+
+	// BilinearPodAutoscalingForAPIServer is applied to a seed cluster and enables simultaneous independent horizontal and vertical scaling.
+	// This feature is incompatible with the HVPA and HVPAForShootedSeed features.
```

Comment:
Should we add a similar text here as the one in `feature_gates.md`?

Reply:
A strictly subjective opinion:
Personally, I'd place a detailed description in the implementing type or package. However I agree with your point. So, WDYT about:
```
	// BilinearPodAutoscalingForAPIServer is applied to a seed cluster and enables simultaneous independent horizontal and vertical scaling.
	// For more information, see the feature_gates.md file.
	// This feature is incompatible with the HVPA and HVPAForShootedSeed features.
```

Reply:
Done. The text from feature_gates.md was duplicated.